examples/cdk2-example/cdk2_setup.yaml

```diff
@@ -0,0 +1,64 @@
+#provide the full path of the protein PDB file
+protein_pdb: CDK2_fixed_nohet.pdb
+
+#provide the path to the ligand file with coordinates
+ligand_file: CDK2_ligands.mol2
+
+#The ligand file contains multiple ligands. Choose the indices of the ligands
+#between which we should compute a relative free energy
+old_ligand_index: 14
+new_ligand_index: 15
+
+#Provide the list of forcefield files. Non-standard (such as gaff.xml) should
+#be provided with a full path
+forcefield_files:
+    - gaff.xml
+    - amber99sbildn.xml
+    - tip3p.xml
+
+#the temperature and pressure of the simulation
+pressure: 1.0
```

Comment:
Let's document the units here with comments for now, and create an issue to later add support for parsing unit-bearing quantities like YANK does. It just uses a [simple function](https://github.com/choderalab/yank/blob/master/Yank/utils.py#L867-L903) to parse the strings.

Reply:
Ok, sounds good.